### DISCUSSION TITLE
146

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#83] [General]
# 潮流周刊第146期 - 桂花小麦

封面图来源于昨晚突发奇想想去喝精酿，点了一杯湖畔桂花小麦，很清爽好喝，特别是第一口很爽。

https://weekly.tw93.fun/posts/146-%E6%A1%82%E8%8A%B1%E5%B0%8F%E9%BA%A6/

**Comment by leon-zym** [comment]
这一期的 `超级全的设计师知识库` 部分和 `把网页转换成 Markdown 格式存储` 部分的标题、内容、图片好像有点对不上。是不是精酿啤酒喝得有点微醺了哈哈哈 😄

**Reply by tw93** [reply]
哈哈哈哈 早上酒醒了 已经改了
